### PR TITLE
[Markdown] [Learn] Remove hidden divs

### PR DESCRIPTION
**Pull request by wbamberg**
Part of https://github.com/mdn/content/issues/9218.

This removes hidden `<div>` elements from the Learn docs. Most of these are associated with live samples, so there was some work needed to give the live samples a heading ID they could refer to.

### CODE REVIEW

File: files/en-us/learn/css/styling_text/styling_links/index.html [hunk 1]
```diff
@@ -281,7 +278,7 @@ <h6 id="Playable_code">Playable code</h6>
 </pre>
 </div>
```
Comment (teoli2003) [inline files/en-us/learn/css/styling_text/styling_links/index.html:279]: I think this `</div>` has to be removed if I checked well.

File: files/en-us/learn/css/styling_text/styling_lists/index.html [hunk 2]
```diff
@@ -362,7 +359,7 @@ <h6 id="Playable_code">Playable code</h6>
 </pre>
 </div>
```
Comment (teoli2003) [inline files/en-us/learn/css/styling_text/styling_lists/index.html:360]: This one too.

File: files/en-us/learn/html/introduction_to_html/getting_started/index.html [hunk 3]
```diff
@@ -187,7 +184,7 @@ <h6 id="Playable_code">Playable code</h6>
 };</pre>
 </div>
```
Comment (teoli2003) [inline files/en-us/learn/html/introduction_to_html/getting_started/index.html:185]: Another `</div>`to remove?

File: files/en-us/learn/html/introduction_to_html/getting_started/index.html [hunk 4]
```diff
@@ -406,7 +400,7 @@ <h6 id="Playable_code2">Playable code2</h6>
...
```
Comment (teoli2003) [inline files/en-us/learn/html/introduction_to_html/getting_started/index.html:401]: Ditto.

File: files/en-us/learn/html/introduction_to_html/getting_started/index.html [hunk 5]
```diff
@@ -653,7 +644,7 @@ <h6 id="Playable_code3">Playable code3</h6>
...
```
Comment (teoli2003) [inline files/en-us/learn/html/introduction_to_html/getting_started/index.html:645]: Dittp

File: files/en-us/learn/html/introduction_to_html/html_text_fundamentals/index.html [hunk 6]
```diff
@@ -222,7 +219,7 @@ <h6 id="Playable_code">Playable code</h6>
 };</pre>
 </div>
```
Comment (teoli2003) [inline files/en-us/learn/html/introduction_to_html/html_text_fundamentals/index.html:220]: Ditto

File: files/en-us/learn/html/introduction_to_html/html_text_fundamentals/index.html [hunk 7]
```diff
@@ -542,16 +533,13 @@ <h6 id="Playable_code_3">Playable code</h6>
...
```
Comment (teoli2003) [inline files/en-us/learn/html/introduction_to_html/html_text_fundamentals/index.html:534]: Ditto

File: files/en-us/learn/html/introduction_to_html/html_text_fundamentals/index.html [hunk 8]
```diff
@@ -696,7 +684,7 @@ <h6 id="Playable_code_4">Playable code</h6>
...
```
Comment (teoli2003) [inline files/en-us/learn/html/introduction_to_html/html_text_fundamentals/index.html:685]: Ditto

File: files/en-us/learn/html/introduction_to_html/html_text_fundamentals/index.html [hunk 9]
```diff
@@ -894,7 +879,7 @@ <h6 id="Playable_code_5">Playable code</h6>
...
```
Comment (teoli2003) [inline files/en-us/learn/html/introduction_to_html/html_text_fundamentals/index.html:880]: Ditto

File: files/en-us/learn/html/multimedia_and_embedding/adding_vector_graphics_to_the_web/index.html [hunk 10]
```diff
@@ -328,7 +325,7 @@ <h6 id="Playable_code">Playable code</h6>
 };</pre>
 </div>
```
Comment (teoli2003) [inline files/en-us/learn/html/multimedia_and_embedding/adding_vector_graphics_to_the_web/index.html:326]: Ditto

File: files/en-us/learn/html/multimedia_and_embedding/images_in_html/index.html [hunk 11]
```diff
@@ -303,7 +300,7 @@ <h6 id="Playable_code">Playable code</h6>
 };</pre>
 </div>
```
Comment (teoli2003) [inline files/en-us/learn/html/multimedia_and_embedding/images_in_html/index.html:301]: Ditto

File: files/en-us/learn/html/multimedia_and_embedding/images_in_html/index.html [hunk 12]
```diff
@@ -478,7 +472,7 @@ <h6 id="Playable_code_2">Playable code 2</h6>
...
```
Comment (teoli2003) [inline files/en-us/learn/html/multimedia_and_embedding/images_in_html/index.html:473]: Ditto

File: files/en-us/learn/html/multimedia_and_embedding/other_embedding_technologies/index.html [hunk 13]
```diff
@@ -185,7 +182,7 @@ <h6 id="Playable_code">Playable code</h6>
 };</pre>
 </div>
```
Comment (teoli2003) [inline files/en-us/learn/html/multimedia_and_embedding/other_embedding_technologies/index.html:183]: Ditto

File: files/en-us/learn/javascript/building_blocks/conditionals/index.html [hunk 14]
```diff
@@ -597,9 +594,9 @@ <h6 id="Playable_code">Playable code</h6>
 };</pre>
 </div>
```
Comment (teoli2003) [inline files/en-us/learn/javascript/building_blocks/conditionals/index.html:595]: Ditto

File: files/en-us/learn/javascript/building_blocks/conditionals/index.html [hunk 15]
```diff
@@ -750,7 +744,7 @@ <h6 id="Playable_code_2">Playable code 2</h6>
...
```
Comment (teoli2003) [inline files/en-us/learn/javascript/building_blocks/conditionals/index.html:745]: Ditto

File: files/en-us/learn/javascript/building_blocks/looping_code/index.html [hunk 16]
```diff
@@ -143,7 +142,9 @@ <h6 id="Hidden_code">Hidden code</h6>
 &lt;/html&gt;</pre>
 </div>
```
Comment (teoli2003) [inline files/en-us/learn/javascript/building_blocks/looping_code/index.html:143]: Ditto

**Review by teoli2003 — CHANGES_REQUESTED** [review]
Nice work; I think you forgot to remove some `</div>`.

**Comment by wbamberg** [comment]
Oops, sorry. Should be fixed now.

**Review by teoli2003 — APPROVED** [review]

**Pull request merged**